library/core/src/iter/traits/iterator.rs

```diff
@@ -3011,6 +3015,18 @@ pub trait Iterator {
     /// assert_eq!(v_cloned, vec![1, 2, 3]);
     /// assert_eq!(v_map, vec![1, 2, 3]);
     /// ```
+    ///
+    /// To get the best performance, try to clone late:
+    ///
+    /// ```
+    /// let a = ["abc".to_string(), "42".to_string(), ".".to_string()];
```

Comment:
can we use `vec![...]` to create some owned thing rather than Strings? Or maybe something else that's concise? I think the `to_string` parts distract from the main point.

Reply:
Done.